linera-storage-service/src/child.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use linera_service::util::CommandExt;
+use std::time::Duration;
+use tokio::process::{Child, Command};
+
+/// Configuration for a storage service running as a child process
+pub struct StorageServiceChild {
+    endpoint: String,
+    binary: String,
+}
+
+/// The tests being done have to run in parallel.
+/// For that we spanned a storage-service per test.
+/// In order to avoid collision, we use a different port per test.
+///
+/// A storage service running as a child process.
+///
+/// The guard preserves the child from destruction and destroys it when
+/// it drops out of scope.
+pub struct ChildGuard {
```

Comment:
The expected name for this would be `StorageService` (very much in line with the code in `linera_service::cli_wrappers::wallet` and the description here).

Reply:
I name it `StorageServiceGuard` which I think expresses the functionality very nicely.